app/controllers/storage_controller/storage_d.rb

```diff
@@ -32,7 +32,7 @@ def storage_list
   def miq_search_node
     options = {:model => "Storage"}
     process_show_list(options)
-    @right_cell_text = _("All %{title} Datastores") % {:title => ui_lookup(:ui_title => "datastore")}
+    @right_cell_text = _("All Datastores") % {:title => ui_lookup(:ui_title => "datastore")}
```

Comment:
@GregP remove setting of :title it is not being used.
the change should be 
```
@right_cell_text = _("All Datastores")
```
or 
```
@right_cell_text = _("All %{title}") % {:title => ui_lookup(:models => 'datastore'}
```